queries.sql

```diff
@@ -8,3 +8,22 @@ SELECT name, escape_attempts from animals where weight_kg > 10.5;
 SELECT * from animals where neutered = true;
 SELECT * from animals where name != 'Gabumon';
 SELECT * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
+
+-- Number of Animals
+select count(*) from animals;
+
+-- How many didn't try to escape
+select count(*) from animals where escape_attempts = 0;
+
+-- Average weight
+select avg(weight_kg) from animals;
+
+-- Who escapes the most
+select max(escape_attempts) from animals;
+select name from animals where escape_attempts = 7;
```

Comment:
- [ ] Kindly make sure to add the query as per the question. The question is `Who escapes the most, neutered or not neutered animals?` You have not added a query for neutered along with escapes the most. Kindly do that.